protos/mission/mission.proto

```diff
@@ -119,5 +119,6 @@ message MissionResult {
         FAILED_TO_OPEN_QGC_PLAN = 9;
         FAILED_TO_PARSE_QGC_PLAN = 10;
         UNSUPPORTED_MISSION_CMD = 11;
+        CANCELLED = 12;
```

Comment:
Wait, does it mean "UPLOAD_CANCELLED"?

Reply:
Or download cancelled.

Reply:
But not `MISSION_CANCELLED`, right? What about `TRANSFER_CANCELLED` then?

Reply:
Right that would be better but you'd have to change it here first:
https://github.com/Dronecode/DronecodeSDK/blob/develop/plugins/mission/include/plugins/mission/mission.h#L54

Reply:
The gRPC server can make the translation until we update the C++ API. Given that the goal will be eventually to generate the C++ API from these proto files, right?